opennms-dao-api/src/main/java/org/opennms/netmgt/dao/api/InterfaceToNodeCache.java

```diff
@@ -50,4 +50,6 @@
 
 	Optional<Integer> getFirstNodeId(String location, InetAddress ipAddr);
 
+	boolean removeInterfacesForNode(int nodeId);
```

Comment:
What's the purpose of the return value here? Looks unused, and always returns `true`.